File: pkg/codegen/dotnet/testdata/throw-not-implemented-pp/throw-not-implemented.cs
using System.Collections.Generic;
using System.Linq;
using Pulumi;

	
object NotImplemented(string errorMessage) 
{
    throw new System.NotImplementedException(errorMessage);
}

return await Deployment.RunAsync(() => 
{
    return new Dictionary<string, object?>
    {
        ["result"] = NotImplemented("expression here is not implemented yet"),
    };
});

File: tests/testdata/codegen/throw-not-implemented-pp/go/throw-not-implemented.go
package main

import (
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
)

func notImplemented(message string) any {
	panic(message)
}

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		ctx.Export("result", notImplemented("expression here is not implemented yet").(pulumi.Any))
		return nil
	})
}
